app/api/apiConfig.js

```diff
@@ -156,7 +156,7 @@ export const settingsAPIs = {
             contact: "telegram:BrianZhang"
         },
         {
-            url: "wss://api.weaccount.cn",
+            url: "wss://api.bts.btspp.io:10100",
```

Comment:
The old one did work for me though. Perhaps we keep both? @syalon what's your opinion?

Reply:
The old one should be maintained until the certificate expires.

Reply:
Got it. We replace it. Thanks.